cdap-docs/reference-manual/source/licenses/index.rst

```diff
@@ -13,12 +13,61 @@ Cask Data Application Platform Licenses and Dependencies
     <cdap-level-1-dependencies>
     <cdap-standalone-dependencies>
 
+Trademarks, Logos, and Trade Dress
+----------------------------------
+
+The following are word trademarks and service marks ("Word Marks", "Service Marks") of
+Cask Data, Inc.:
+
+.. list-table::
+   :widths: 20 80
+   :header-rows: 1
+
+   * - Mark
+     - Common descriptive name for the goods or services
+   * - Cask
+     - Computer software and related services
+   * - CDAP
+     - Computer software and related services
+   * - Coopr
+     - Computer software and related services
+   * - Coopr Cloud
+     - Computer software and related services
+   * - Tephra
+     - Computer software and related services
+   * - Tigon
+     - Computer software and related services
+
+
+Our logos ("Logos") include any logos or illustrations used on these webpages.
+The unique visual styling of our website and packaging is referred to as the "Trade Dress".
+
+This encompasses all trademarks and service marks, whether Word Marks, Logos or Trade
+Dress, whether registered or not, which are collectively referred to as the “Marks.” 
+
+All marks are trademarks of Cask Data, Inc. All rights reserved.
+
+
+Other Products and Trademarks
+-----------------------------
+
+Other product and company names mentioned herein, within our documentation or webpages, may
+be the trademarks of their respective owners. Where the ownership or registration is known,
+the addition of appropriate symbols has been made.
+
+Apache®, Apache Hadoop, and Hadoop® are either registered trademarks or trademarks of the
```

Comment:
Though now that I re-read it, they suggest adding hyperlinks. Which I will do...